### DISCUSSION TITLE
ExplorerPatcher Weather glitching out

### DISCUSSION THREAD

**Discussion by amanas-us** [valinet/ExplorerPatcher#1352] [General]
Is anyone else having the problem where the weather on the taskbar doesn't work right? For me, it shows the weather then says, 'you're not connected'. @valinet help!
I will upload a video file here:

https://user-images.githubusercontent.com/125621033/233514805-7aaea2c1-bb6e-4a1e-a964-6ab047fb0c40.mp4

**Comment by topia** [comment]
Why did you duplicate this as the issue? Anyway, it's duplicated problem; please close this.
